handbook.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: handbook
+title: Handbook to organize a Gallantries event
+---
+
+We're thrilled you choose to run a Gallantries event.
+
+We hope this guide will help you navigate the logistics of the event organization. If anything is unclear or you have more questions, feel free to add comments in [GitHub]({{ site.github.repository_url }}/issues/new), or reach out to us at [{{ site.email|replace:'@','[at]' }}](mailto:{{ site.email }}) directly. We're here to help!
+
+# What is a Gallantries event?
+
+Several recent surveys highlight the need and demand from early-stage researchers for training in management and analysis of their High Throughput Sequencing (HTS) data. Given the gap in the education currently offered by traditional providers, there is a distinct push to create and provide high quality, decentralized, accessible, scalable and practical training in bioinformatics.
+
+The Gallantries team aims to address this gap by preparing training material for the computational analysis of HTS data, starting from the raw data and leading up to the development of publication ready visualizations of the analyzed results, using the Galaxy platform as the underlying infrastructure. This is a representative training scenario as it covers themes and practices common in the vast majority of data analysis pipelines in life sciences: 
+
+1. **User-friendly data analysis in Galaxy**, from the raw sequencing data to the downstream count tables
+2. **Downstream analysis & visualisation using Rstudio in Galaxy** with manipulation and visualization of the results within R, including an introduction to R using RStudio
+
+Beyond the curriculum, Gallantries team aims to address the scalability of training delivery via **hybrid training events**, i.e. pairing-up on-site helpers with remote instructors across multiple sites simultaneously:
+
+![](assets/images/handbook/hybrid_training.png)
+
+With this model, we aim to bring training events to the trainers while reducing the environmental impact of instructor travels.
+
+# A typical Gallantries hybrid training event
+
+*To illustrate a typical Gallantries event and the different roles (1 person can have several roles), we decided to share a story with personna*
+
+**Max** is a training coordinator of an international research consortium. They recently surveyed researchers in the consortium on their need in HTS data analysis. The results were overwhelming: over 60 scientists on several locations would like to learn how to analyze their own RNA-seq data. To fulfill this demand, Max decided to organize a Gallantries event: a 3-days workshop with 4 different locations simultaneously (Greece, Estonia, France and Spain). Max is the **global organizer** of this event, in charge to find the date, contact and coordinate with the local hosts, advertize the event, find the instructors, etc. After the event, Max will aggregate all feedback from participants, helpers and hosts and share them.
```

Comment:
```suggestion
**Max** is a training coordinator of an international research consortium. They recently surveyed researchers in the consortium on their need in HTS data analysis. The results were overwhelming: over 60 scientists at several locations would like to learn how to analyze their own RNA-seq data. To fulfill this demand, Max decided to organize a Gallantries event: a 3-days workshop with 4 different locations (Greece, Estonia, France and Spain). Max is the **global organizer** of this event, in charge to find the date, contact and coordinate with the local hosts, advertize the event, find the instructors, etc. After the event, Max will aggregate all feedback from participants, helpers and hosts and share them.
```

on several → at several, removed simultaneously since too verbose + clear from previous parts.